demos-europe/demosplan-core
# feat: use avscan http service

Provide an implementation to an external Viruscheck service via http that implements the interface in #2506 

### How to review/test
the containers will be included into the developmentsetup soon. To check it beforehand you may add the following services to your `docker-compose.override.yml`
```yaml
    clamd:
        image: clamav/clamav:0.104
        restart: unless-stopped
        networks:
            demosplan_development_network:
                ipv4_address: 172.22.255.99
    avscan_api:
        image: benzino77/clamav-rest-api
        restart: unless-stopped
        depends_on:
            - clamd
        environment:
            - NODE_ENV=production
            - CLAMD_IP=clamd
            - APP_FORM_KEY=FILES
            - APP_PORT=3000
            - APP_MAX_FILE_SIZE=2147483648
        networks:
            demosplan_development_network:
                ipv4_address: 172.22.255.100

```

### Linked PRs (optional)


### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
